templates/torrc-sample

```diff
@@ -76,4 +76,8 @@ HiddenServicePort 80 <app-samourai-server-ip>:80
 HiddenServiceDir /var/lib/tor/app-samourai-server-whirlpool
 HiddenServicePort 80 <app-samourai-server-whirlpool-ip>:<app-samourai-server-whirlpool-port>
 
+# LndHub Hidden Service
+HiddenServiceDir /var/lib/tor/app-lndhub
+HiddenServicePort 3000 <app-lndhub-ip>:<app-lndhub-port>
```

Comment:
Is it required that the user connect via `http://xxxx.onion:3000/` when using tor?

All the other apps use 80 to eliminate the port from the URL, but does LNDHub need it?

Reply:
if thats a question about bw - bw can use any port

Reply:
Nope @Overtorment, it's not about BW internal port, but the port used for the app Tor hidden service. 

Usually, we set it to 80 so the onion hostname is sufficient to connect, but here it is set to 3000, so the user would need to enter blabla.onion:3000 instead of blabla.onion.